docs/website/contents/for-developers/ProtocolVersioning.md

```diff
@@ -0,0 +1,155 @@
+# Introduction
+
+The Cardano protocol will change over time.
+The community innovates applications and updates feature priorities, researchers refine Ouroboros, and developers improve the code.
+
+This document describes how Cardano accomodates these changes as a community, as a software engineering project, and as a distributed system.
+
+## The Protocol Version
+
+Each change to the Cardano protocol advances through the following steps before influencing the network's chain.
+
+- Someone (you!) specifies a change that is backwards-incompatible but worthwhile.
+  For example, it could be a [Cardano Improvement Proposal](https://cips.cardano.org/) (CIP).
+- The Technical Steering Committee and Ouroboros researchers address the worth, safety, and future consequences of the proposed solution.
+- Cardano developers prepare a software release that includes the changes but only enables them for chains on which the protocol version has been incremented.
+- The community uses on-chain governance to increment the protocol version, thereby putting the change into effect.
+  (Stake pool operators must not vote to increment the protocol version before updating their software!)
```

Comment:
Is it sufficiently clear for the reader which protocol version increment we're referring to? I guess this refers to the protocol version included in the headers, which denotes the maximum protocol version our node supports.